server/src/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -3242,8 +3283,9 @@ protected UserVm createVirtualMachine(DataCenter zone, ServiceOffering serviceOf
             throw new InvalidParameterValueException("Installing from ISO requires an ISO that is bootable: " + template.getId());
         }
 
-        // Check templates permissions
-        _accountMgr.checkAccess(owner, AccessType.UseEntry, false, template);
+        // Check templates permissions when the create vm is not from snapshot
```

Comment:
@yvsubhash Here is the code snippet picked up from PR https://github.com/apache/cloudstack/pull/1721